helm/hpcc/values.yaml

```diff
@@ -278,6 +288,13 @@ bundles: []
 # for example
 # - name: DataPatterns
 
+dafilesrv:
+- name: mydafilesrv
+  disabled: true # disabled by default because requires cert-manager etc. (see certificates section)
+  service:
+    servicePort: 7600
```

Comment:
Do we have to list 7600 in thress places ?

Reply:
it's useful to list here for doc. purposes, but as it's the default it could be commented out here.
I'll do that.

Reply:
actually, I am going to leave as it was, because the service definition is shared an in common with other similar service definitions the schema requires it to be present, so will keep it here too for consistency.